.github/workflows/generate-postman-collection.yml

```diff
@@ -29,6 +29,5 @@ jobs:
             git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
Ook hiervoor geldt volgens mij dat de master branch versie van dit bestand nieuwer is dan die in de develop.
@MelvLee Klopt dat?

Reply:
klopt. Dat is nu gefixt met een merge vanuit main